lib/hawkular/alerts/alerts_api.rb

```diff
@@ -14,7 +14,7 @@ module Hawkular::Alerts
   # @param credentials [Hash{String=>String}] Hash of username, password, token(optional)
   # @param options [Hash{String=>String}] Additional rest client options
   class AlertsClient < Hawkular::BaseClient
-    def initialize(entrypoint = 'http://localhost:8080/hawkular/alerts', credentials = {}, options = {})
+    def initialize(entrypoint, credentials = {}, options = {})
```

Comment:
What happens if one does `AlertsClient.new('localhost:8080', credentials, options)` ? I think that in this case the constructor would need to internally attach the `/hawkular/alerts` part